### PR TITLE
Task-42873: Change the published news detail page when accessible by non space members

### PR DESCRIPTION
**Pull request by hassenelkamel-exo**
**Problem:**  When displaying published article for non space member, the tabs content  is `News details`.
**Solution:**  When getting the news to be displayed, we set the right document title `News: title of article`

### CODE REVIEW

**Review by azayati — APPROVED** [review]
LGTM after QA validation

File: webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue [hunk 1]
```diff
@@ -289,6 +289,7 @@ export default {
           return this.$nextTick();
         })
         .finally(() => {
+          document.title = this.$t('news.window.title', {0: this.news.title});
```
Comment (azayati) [inline webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue:292]: Should you ensure that you display only the 100 first chars of the news title ?

**Review by azayati — CHANGES_REQUESTED** [review]
See review

File: webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue (hunk 1 above)
Comment (hassenelkamel-exo) [reply webapp/src/main/webapp/news-details/components/ExoNewsDetails.vue:292]: It display the same like the activity

**Review by azayati — APPROVED** [review]
LGTM

**Pull request merged**